HOWTO.md

```diff
@@ -523,10 +524,41 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    `/etc/fdo/manufacturing-serverr.conf.d/`. The paths will be checked in that
    same order.
 
-3. Execute `fdo-manufacturing-server` or run it as a service, see sample
-   file in
+3. Execute `fdo-manufacturing-server` using below command:
+   ```
+   sudo MANUFACTURING_SERVER_CONF=/usr/share/fdo/manufacturing_server.yml LOG_LEVEL=trace ./target/debug/fdo-manufacturing-server
+   ```
+   or run it as a service, see sample file in
    [examples/systemd](https://github.com/fedora-iot/fido-device-onboard-rs/blob/main/examples/systemd/fdo-manufacturing-server.service).
 
+### Manufacturing Client
+
+1. Once you have generated required keys/certificates for the Manufacturing Server and manufacturing-server is running
+   (Refer above section: Manufacturing Server) you can run manufacturing-client. Use case such as: if you need to check 
+   DI(Device Initialization)/ DIUN(Device Initialization over Untrusted Network) protocol is working as expected, then you
+   can run manufacturing-server and manufacturing-client on their own. 
+2. If you are running plainDI, set following environmental variables before running manufacturing client:
+    - DI_MFG_STRING_TYPE = SerialNumber/SerialNumberExt
+    - DI_KEY_STORAGE_TYPE = FileSystem/ Tpm 
+    - DI_SIGN_KEY_PATH = /path/to/keys
+    - DI_HMAC_KEY_PATH = /path/to/keys
+3. If you are running DIUN use below command : 
+   ```
+   sudo MANUFACTURING_SERVER_URL=http://localhost:8080 LOG_LEVEL=trace DEVICE_CREDENTIAL=dest-dir/device-credentials DIUN_PUB_KEY_ROOTCERTS=aio-dir/keys/diun_cert.pem ./target/debug/fdo-manufacturing-client
+   ```
+   In this case mfg_string_type is passed from manufacturing-server to manufacturing client during DIUN ,which comes from manufacturing-server.yml config file. You need to specify under diun settings e.g.
+   ```
+   diun:
+   mfg_string_type: SerialNumber (or) mfg_string_type: SerialNumberExt
```

Comment:
This configuration is part of the Manufacturing server, there is already a section that explains this so you should update that section with the new `SerialNumberExt`.